multisig/README.md

```diff
@@ -54,6 +64,8 @@ Guarantees:
 ### Gotchas
  
 User can delete access keys such that total number of different access keys will fall below `num_confirmations`, rendering contract locked.
+
+`AddKey` on another account adds full access key.
```

Comment:
Actually in the other PR added optional permissions to allow non full access keys